app/ModelFunctions/PhotoFunctions.php

```diff
@@ -234,7 +259,9 @@ public function add(array $file, $albumID_in = 0)
 		// Set paths
 		$tmp_name = $file['tmp_name'];
 		$photo_name = md5(microtime()) . $extension;
-		$path = Storage::path('big/' . $photo_name);
+
+		$path_prefix = $kind != 'RAW' ? 'big/' : 'raw/';
```

Comment:
I don't know what I'd do instead, but it feels strange hard coding this in several places.